packages/transitive-overlay/src/leaflet-canvas-layer.js

```diff
@@ -119,6 +119,7 @@ L.CanvasLayer = (L.Layer ? L.Layer : L.Class).extend({
     //------------------------------------------------------------------------------
     drawLayer: function () {
         // -- todo make the viewInfo properties  flat objects.
+        if (!this._map) return
```

Comment:
Insert `;` after `return`.

Reply:
So easy to forget the luxuries of an eslint environment. Fixed in 9ad268ac61f7db821681691dd134176a5b3bba75!